syntax = "proto3";

import "shiva/core/communication_objects/helpers.proto";

package communication_objects;

service IOHandler {
    rpc AskIORequest(SimpleMessage) returns (SimpleMessage);
    rpc DoneIO(SimpleMessage) returns (Empty);
//    rpc SendSpecs(SimpleMessage) returns (Empty); // used for the single environments to check-in
//    rpc GetSpecs(SimpleMessage) returns (SimpleMessage);
//    rpc GetActions(SimpleMessage) returns (SimpleMessage);
//    rpc SendConfig(ConfigProto) returns (Empty);
}